pootle/apps/pootle_fs/management/commands/init_fs_project.py

```diff
@@ -94,11 +93,15 @@ def handle(self, **options):
             project = Project.objects.create(
                 code=code,
                 fullname=name,
-                localfiletype=options['file_type'],
                 treestyle='none',
                 checkstyle=options['checkstyle'],
-                source_language=source_language
-            )
+                source_language=source_language)
+            for filetype in options["filetypes"] or ["po"]:
```

Comment:
it was my lack of argparse-fu - it was failing every way i tried it - apart from this